.github/workflows/test.yml

```
@@ -2,7 +2,7 @@ name: test
 
 on:
   push: # Run on pushes to the default branch
-    branches: [main]
+    branches: [main, 448-remote-manifest-support]
```

as suggested in previous comment https://github.com/astronomer/astronomer-cosmos/pull/1109#discussion_r1687979238, duplicating the comment so that it appears well before merging.
```suggestion
    branches: [main]
```